config/application.rb

```diff
@@ -11,5 +11,15 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+
+  ActionMailer::Base.smtp_settings = {
```

Comment:
remove this please, we have already set up the mailer for development on config/environments/development.rb